src/outputs/restart.hpp

```diff
@@ -180,6 +179,9 @@ class RestartReader {
   // perhaps belongs in a destructor?
   void Close();
 
+  // Does file have ghost cells?
+  int hasGhost;
```

Comment:
Why int instead of bool?

Reply:
Because it is read in from the HDF file and this requires fewer lines of code